packages/main/src/plugin/extension-loader.ts

```diff
@@ -962,8 +962,8 @@ export class ExtensionLoader {
       ) {
         return containerProviderRegistry.buildImage(context, eventCollect, options);
       },
-      listImages(): Promise<containerDesktopAPI.ImageInfo[]> {
-        return containerProviderRegistry.listImages();
+      listImages(options?: containerDesktopAPI.ListImagesOptions): Promise<containerDesktopAPI.ImageInfo[]> {
+        return containerProviderRegistry.listImages(options);
```

Comment:
we should add a test in extension-loader.spec.ts

Reply:
Done ✅